server/auto_updater/auto_updater_runner.py

```diff
@@ -0,0 +1,87 @@
+from dataset_updater import (
+    gather_new_papers,
+    generate_vector_counts,
+    merge_files,
+    update_dictionaries,
+)
+
+from embedding_bin_updater import (
```

Comment:
Did you forget to add `embedding_bin_updater.py` to this PR? I can't find this module.